### DISCUSSION TITLE
[BUG]

### DISCUSSION THREAD

**Discussion by boxzhang88** [1Panel-dev/1Panel#6292] [Q&A]
### 1Panel 版本

v1.10.13-lts

### 问题描述

配置文件一 docker-compose.yml 中有nginx 服务 网络属于网络A、B、配置文件二docker-compose.yml 有个gateway 服务也在网络A、B中。启动这两个文件可以正常通讯。但是当通过页面点击重启nginx，结果配置文件一中nginx和配置文件二中gateway 服务 不能通讯了。  排查发现 ：nginx只属于A 网络了。B网络中没有nginx了。我的解决办法是，去掉B网络。两个配置文件网络全用A 网络。（桥接网络）

### 重现步骤

![22222](https://github.com/user-attachments/assets/19d68d31-b71a-4d26-8fc4-921339271e8e)

### 期待的正确结果

修复重启后网络环境修改问题。

**Comment by wanghe-fit2cloud** [comment]
麻烦提供一下两个配置文件，我们本地环境复现下这个问题。
